src/main/java/smartthings/konsumer/stream/ThreadedMessageConsumer.java

```diff
@@ -43,18 +43,18 @@
 
 	public ThreadedMessageConsumer(
 			KafkaStream<K, V> stream, Executor messageExecutor, ListenerConfig config,
```

Comment:
I don't think the ListenerConfig is used anymore with this change. So we no longer need to pass it in to the MessageConsumer